src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.html

```diff
@@ -44,6 +44,42 @@
           </div>
         </div>
 
+        <!-- RGW Export Type -->
+        <ng-container>
+          <div *ngIf="storageBackend === 'RGW' && !isEdit"
+               class="form-group row">
+            <label class="cd-col-form-label required"
+                   for="rgw_export_type"
+                   i18n>Type</label>
+            <div class="cd-col-form-input">
+              <div class="form-check form-check-inline">
+                <input class="form-check-input"
+                       formControlName="rgw_export_type"
+                       id="bucket_export"
+                       type="radio"
+                       name="rgw_export_type"
+                       value="bucket"
```

Comment:
can't we just use formControlName here ?

Reply:
I think we'll need `id` & `value` atleast for radio buttons, isn't it?